var/spack/repos/builtin/packages/python/package.py

```diff
@@ -148,6 +148,15 @@ def install(self, spec, prefix):
         make()
         make('install')
 
+        self.sysconfigfilename = '_sysconfigdata.py'
+        if spec.satisfies('@3.6:'):
```

Comment:
I don't know if there is a good URL describing the change. I just noticed the file name changed when some of our users reported a build problem with our 3.6.0 python.